packages/vite/src/rsc/rscBuildAnalyze.ts

```diff
@@ -61,19 +61,16 @@ export async function rscBuildAnalyze() {
     build: {
       manifest: 'rsc-build-manifest.json',
       write: false,
-      ssr: true,
+      // TODO (RSC): In the future we want to generate the entries file
+      // automatically. Maybe by using `analyzeRoutes()`
+      // For the dev server we might need to generate these entries on the
+      // fly - so we will need something like a plugin or virtual module
+      // to generate these entries, rather than write to actual file.
+      // And so, we might as well use on-the-fly generation for regular
+      // builds too
+      ssr: rwPaths.web.entries,
```

Comment:
The Vite docs say we can just make this a string inline here instead of needing another config block so seeing if that'll go. See https://vitejs.dev/config/build-options#build-ssr.